src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
+}
+
+h1 {
+  margin: 0;
+  font-size: 17px;
+  font-weight: bold;
```

Comment:
this is default value in h elems

Reply:
fixed it although I don't think that it is bug.